website/content/en/docs/deployment/cognito-rds-s3/guide-terraform.md

```diff
@@ -73,14 +75,31 @@ pwd
     export USE_RDS="true"
     # true/false flag to configure and deploy with S3
     export USE_S3="true"
+    # Pipeline S3 Credential Option to configure 
```

Comment:
Since you have the section below was this meant to be removed?

Reply:
looks like missed removing the comment